index.html

```diff
@@ -141,8 +194,13 @@ <h3 class="title">Multi-Post Stories Gain+Glory</h3>
     <h2 class="about-me-heading">About Me</h2>
 </div>
 
-    <p class="animate-para"> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
-    don’t hesitate to contact me.</p>
+    <p> 
+      I can help you build a product, feature or website. Look through some of my work
+      <br>
+       and experience! If you like what you see and have a project you need coded, 
+       <br>
+      don’t hesitate to contact me.
+    </p>
```

Comment:
- [ ] I kindly suggest that you ensure this paragraph does not exceed 3 to 4 lines on the desktop as this is part of the project requirements. please fix this as your paragraph text is currently 5 lines on desktop